framework/console/controllers/BaseMigrateController.php

```diff
@@ -292,7 +326,10 @@ public function actionRedo($limit = 1)
      */
     public function actionTo($version)
     {
-        if (preg_match('/^m?(\d{6}_\d{6})(_.*?)?$/', $version, $matches)) {
+        if (preg_match('/^\\\\?([\w_]+\\\\)+m(\d{6}_?\d{6})(\D.*?)?$/is', $version, $matches)) {
```

Comment:
If `\` is trimmed afterwards it makes sense to allow any number of slashes before and after.